editor/js/ui/projects.js

```diff
@@ -694,6 +735,90 @@ RED.projects = (function() {
         return container;
     }
 
+    $.fn.isVisible = function() {
```

Comment:
I would prefer not to patch functions onto `$` in the middle of the code base - otherwise random things could be added anywhere and it will be impossible to manage and avoid duplication.

In this instance, I believe this functionality can be replaced with `deleteButton.is(":visible")` - although there are some edge cases that doesn't work on.